### ISSUE TITLE
add a new instrument key value pair object

### ISSUE THREAD

**Issue by Maigard** [pacifica/pacifica-metadata#197]
Make a connection between key value pairs and instruments and analytical tools. 

create and document endpoint.

**Comment by dmlb2000** [comment]
These key value pairs in this table would be used to express required key/value pairs for an instrument or analytical tool to upload data. As this has policy server implications referencing https://github.com/pacifica/pacifica-policy/issues/64 for that issue.

**Comment by dmlb2000** [comment]
I've got this merged into master by #206 this unblocks the policy effort to build the API.
